Lab2.ipynb

```diff
@@ -0,0 +1,677 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = np.linspace(0, 10, 1000)\n",
+    "dist = stats.rayleigh.pdf(x)\n",
+    "\n",
+    "conv1 = np.convolve(dist, dist)\n",
+    "x1 = np.linspace(0, len(conv1), len(conv1))\n",
```

Comment:
Also in 2A, `x1` should be defined as `np.linspace(0, 10, len(conv1))`, because when you take the average of two distributions the range of values which you're considering shouldn't change.